src/health/config/custom-environment-variables.json

```diff
@@ -5,13 +5,14 @@
   "apikeyHost": "APIKEY_HOST",
   "mailHost": "MAIL_HOST",
   "elasticHost": "ELASTIC_HOST",
-  "unpaywall": {
-    "host": "UNPAYWALL_HOST",
-    "apikey": "UNPAYWALL_APIKEY"
-  },
   "redis":  {
     "host": "REDIS_HOST",
     "port": "REDIS_PORT",
     "password": "REDIS_PASSWORD"
+  },
+  "elastic": "ELASTIC_HOST_LOGIN",
```

Comment:
ELASTIC_HOST_LOGIN ? Is that supposed to be the elastic user ?

Reply:
this variable contains host, login, password